Slam/Queue/QueueView.swift

```diff
@@ -2,12 +2,18 @@ import UIKit
 
 public class QueueView: NSObject, UITableViewDataSource {
     let display: UITableView
+    public var onDelete: (String) -> Void = { arg in }
     public var matches: [Match] = []
 
     public init(display: UITableView) {
```

Comment:
Should this initializer go away? Is there any ever time that we would not pass in an `onDelete`?